test/unit.jl

```diff
@@ -1,4 +1,19 @@
 @testset "unit tests" begin
+
+    @testset "tmpdir" begin
+
+        #=
+        Leftover files from failed tests or errors can cause testing errors.
+        Therefore, we remove any files in the `images` directory not pertinent to testing before executing a test.
+        =#
+
+        EXCLUDE_FILES = [".keep"]
+        for f in readdir("images")
+            !(f in EXCLUDE_FILES) && rm("images/$f")
+        end
+        @test length(readdir("images")) == length(EXCLUDE_FILES)
+    end
```

Comment:
This shouldn't appear here. I think you might have messed something up with your base branch. This should go into v0.2

Reply:
I reverted this to v0.2 - not sure what happened? Might've been when I accidentally started off of master branch.